interfaces/seccomp/backend.go

```diff
@@ -81,6 +83,19 @@ func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions,
 	if err != nil {
 		return fmt.Errorf("cannot synchronize security files for snap %q: %s", snapName, err)
 	}
+
+	for baseName := range content {
+		in := filepath.Join(dirs.SnapSeccompDir, baseName)
+		out := filepath.Join(dirs.SnapSeccompDir, baseName+".bpf")
```

Comment:
Problem with not having .bin is that everywhere else those are text files, and tab-completing them will hit the naked file before the .src, so people will tend to have an editor full of garbage. The .src and .bin make it very easy to grasp what's the content in each case without having to open it up.